manager/assets/modext/widgets/system/modx.panel.dashboard.js

```diff
@@ -324,7 +324,7 @@ Ext.extend(MODx.window.DashboardWidgetPlace,MODx.Window,{
             fld.markInvalid(_('dashboard_widget_err_placed'));
             return false;
         }
-        var rank = s.getTotalCount();
+        var rank = s.data.length;
```

Comment:
As far as i remind, ranks should be starting at 0, which should make the `+ 1` useless.
All in all, when ranks are messed up, increment on the number of records is not the solution (we should rather try to find the highest rank then increment that one).

I might triple check that